sparse/boot/flash-on-windows.bat

```diff
@@ -87,6 +87,45 @@ exit /b 1
 :: Now we know which device we need to flash
 set fastbootcmd=%fastbootcmd_no_device% -s %current_device%
 
+:: Verify that the Sony release on the phone compatible.
+@call :getvar version-baseband
+
+:: Take from e.g. 1311-2918_50.1.A.13.123 the first number set, i.e., 50.1
+for /f "tokens=2 delims=_" %%i in ('type %tmpflashfile%') do @set version1=%%i
+for /f "tokens=1,2,5 delims=." %%a in ('echo %version1%') do (
+  @set vmajor=%%a
+  @set vminor=%%b
+  @set vpatch=%%c
+)
+
+:: We only support devices that have been flashed with Android 8.1
+:: 50.1.x.x.x of the Sony Android delivery.
+@set rmajor=50
+@set rminor=1
+
+if %vmajor% GTR %rmajor% goto :error_too_new_version
+if %vmajor% LSS %rmajor% goto :version_ok
+
+if %vminor% GTR %rminor% goto :error_too_new_version
+if %vminor% LSS %rminor% goto :version_ok
+
+goto :version_ok
+
+:error_too_new_version
+echo(
+echo Your Sony Android version on your device is too new (Android 9 or later).
+echo You need to have Android 8.1 in order for this installation to work.
+echo(
+echo Unfortunately any known methods to downgrade to Android 8.1 will make your device no longer bootable.

```

Comment:
At least I have problems grokking what this sentence tries to say... could use rewording?

(the same text is in shell script too)

Reply:
Was xperiaflashtool tested?
Thats unofficial boot works because it reset the whole firmware and not just android.

Reply:
Message from the affected party: "And I also tried the flashtool for Linux - same results."